src/filter/mod.rs

```diff
@@ -1,3 +1,5 @@
+/// aHash implementation
+pub mod ahash;
```

Comment:
I think, it is better to remove `pub` for this module. `filter` module is public, so this gives an access to `ahash` impl, which should be private